README.md

```diff
@@ -134,6 +134,12 @@ bundle exec rake 'schools:send_invites[urn1 urn2 ...]'
 bundle exec rake lead_provider:generate_token "name or id"
 ```
 
+## Run payment calculator for a given lead provider to generate the payment breakdown
+
+```bash
+bundle exec rake payment_calculation:breakdown "name or id"
```

Comment:
Probably warrants an update now that we added number of participants

Reply:
Thanks, done in c6e947e